src/components/Map/Map.js

```diff
@@ -26,13 +26,32 @@ const Map = withScriptjs(withGoogleMap(props => (
     {...props}
     ref={props.mapRef}
   >
+
+      { props.isWalkArlingtonVisible && <KmlLayer
+          url="https://raw.githubusercontent.com/microcat49/KMLFiles/master/Copy%20of%20WalkBoston%20-%20Arlington.kml"
```

Comment:
Could  we make these be relative URLs and bundle them into code deployment? Alternatively - is there a more canonical place these could be hosted?

Reply:
Unfortunately I don't think we can make these relative URLs. Just because in order to pull these off of github we need the "raw.githubusercontent.com". I also was not able to pull these off of localhost.